src/components/Proof/Testimonials.jsx

```diff
@@ -0,0 +1,34 @@
+import { feedback } from "./constants/index";
+import FeedbackCard from "./FeedbackCard";
+
+const Testimonials = () => (
+  <section
+    id="clients"
+    data-testid="testimonials-section"
+    className="sm:py-5 py-4 flex justify-center items-center flex-col"
+  >
+
+    <div className="w-full flex justify-between items-center md:flex-row flex-col sm:mb-4 mb-2">
+      <h2 className="font-Oxygen font-bold text-center xs:text-[48px] text-[40px] text-black xs:leading-[76.8px] leading-[66.8px] w-full"
```

Comment:
we need to use the font we configured in tailwind-config file, so please remove the font family here .